deliverables/README.md

```diff
@@ -31,38 +36,154 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+They are separated for organization (meaning) and clarity (presentation). 
```

Comment:
Actually organization and clarity are the same thing.  Putting styles all mixed with html is fine for the user, but unclear for the developer.  We organize them (specialize the pieces into different places with different functions) in order to make it more clear for the developer.  90% of what coders do is keeping the code well-organized and hence easily readable by other coders (and themselves)